src/views/TabView/TabBarBottom.js

```diff
@@ -71,10 +71,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
       inputRange,
       outputRange,
     });
+    const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
@Ehesp No string is fine. If a custom component is specified, we should just return it without wrapping in a view.

Also `labelStyle` is for a text label, so you can't apply them to a view.